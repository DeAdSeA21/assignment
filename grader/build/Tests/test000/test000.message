Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t3910.53
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
