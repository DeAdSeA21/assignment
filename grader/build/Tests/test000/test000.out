Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	9543.75
2	9085.50
3	8625.25
4	8162.99
5	7698.70
6	7232.38
7	6764.03
8	6293.62
9	5821.15
10	5346.62
11	4870.01
12	4391.32
